modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/domain/DefaultDomainFactoryTestCase.java

```diff
@@ -84,20 +77,21 @@ public class DefaultDomainFactoryTestCase extends AbstractDomainTestCase {
                                                                               getRuntimeLockFactory(),
                                                                               mock(MemoryManagementService.class),
                                                                               mock(ArtifactConfigurationProcessor.class));
+  private MockedStatic<AbstractDeployableProjectModelBuilder> utilities;
 
   public DefaultDomainFactoryTestCase() throws IOException {}
 
   @Before
   public void setUp() throws Exception {
-    mockStatic(AbstractDeployableProjectModelBuilder.class);
-    when(isHeavyPackage(any())).thenReturn(true);
-    when(defaultDeployableProjectModelBuilder(any(), any(), anyBoolean())).thenCallRealMethod();
-    DeployableProjectModel deployableProjectModelMock = PowerMockito.mock(DeployableProjectModel.class);
+    utilities = mockStatic(AbstractDeployableProjectModelBuilder.class);
```

Comment:
try with resources without the @After

Reply:
@fsgonz If I use try-with-resource mechanism in the setup method, tests are failing. Would it be better to put this try scope in each tests then?

Reply:
It's ok as it is.

Reply:
It's ok as it is.